common/src/main/java/com/wynntils/models/statuseffects/type/StatusEffect.java

```diff
@@ -84,4 +105,23 @@ public int compareTo(StatusEffect effect) {
                 .compare(this.getModifier().getString(), effect.getModifier().getString())
                 .result();
     }
+
+    private Pair<StyledText, Optional<Double>> readModifierValueAndSuffix(StyledText modifier) {
+        String modifierStr = modifier.getStringWithoutFormatting();
+        if (modifierStr.isEmpty()) {
+            return new Pair<>(StyledText.EMPTY, Optional.empty());
+        }
+        int start = Math.max(modifierStr.indexOf('+') + 1, 0);
```

Comment:
Technically if there is no +, index is -1 + 1 is always 0. You can remove the max and comment this.